dev_install.adoc

```diff
@@ -0,0 +1,99 @@
+= Dev Sandbox Development Install
+
+This document describes how to install Dev Sandbox in a development environment.
+
+== Prereqs
+
+=== OpenShift Cluster
+Ensure you have access to an OpenShift 4.6+ cluster with cluster admin privileges and log in using `oc login`
+
+=== Required Tools
+Install the link:required_tools.adoc[required tools].
+
+=== Quay
+. Register for a quay.io account if you don't have one
+. Make sure you have set the `QUAY_NAMESPACE` variable: +
+`export QUAY_NAMESPACE=<quay-username>`
+. Log in to quay.io using `docker login quay.io`
+. Make sure that the visibility of the `host-operator`, `member-operator`, `member-operator-webhook` and `registration-service` repositories on quay.io are all set to `public`:
+ * https://quay.io/repository/<your-username>/host-operator?tab=settings
+ * https://quay.io/repository/<your-username>/member-operator?tab=settings
+ * https://quay.io/repository/<your-username>/member-operator-webhook?tab=settings
+ * https://quay.io/repository/<your-username>/registration-service?tab=settings
+
+=== Authentication
+Configure authentication for the cluster using one of the following options: +
+ +
+Option #1: Contact a member of the Dev Sandbox Team for instructions on how to configure the cluster to use our internal Dev SSO. +
+ +
+Option #2: Configure your own Keycloak server and set up authentication on the OpenShift cluster: https://docs.openshift.com/container-platform/4.6/authentication/configuring-internal-oauth.html
+
+== Install
+
+. Clone this repository +
+`+git clone git@github.com:codeready-toolchain/toolchain-e2e.git+`
+. Run the following to install the Dev Sandbox operators +
+`make dev-deploy-e2e`
+. Make note of the Registration Service URL that is printed at the end
+. Run `oc get toolchainstatus -n <quay-io-username>-host-operator` and ensure the Ready status is `True`
++
+```
+NAME               MURS   READY   LAST UPDATED
+toolchain-status   0      True    2021-03-24T22:39:36Z
+```
+
+. Open the Registration Service URL in a browser and sign up for an account.
+
+. Wait for the message "Your OpenShift Developer Sandbox account is waiting for approval"
+
+image::https://user-images.githubusercontent.com/20015929/114627893-01845d00-9c84-11eb-848e-0f85a1b3c01f.png[]
+
+=== Approve The User
+
+==== Manual Approval
+
+Manual approval means each usersignup must be approved by editing the usersignup resource for a particular user.
+
+. Approve the usersignup
+```
+oc patch usersignup -p '{"spec":{"approved":true}}' --type=merge <usersignup name> -n <host operator namespace>
```

Comment:
Maybe worth mentioning how to get <usersignup name> and what it is. Just to make it clear that it's not the username.

Reply:
Done https://github.com/codeready-toolchain/toolchain-e2e/pull/280/commits/8d38de5ee123984bca02d9ce764fbeccf823c7a3